src/caffeinated_whale_cli/main.py

```
@@ -49,21 +52,26 @@ def main(
 app.add_typer(restart_cmd.app, name="restart")
 app.add_typer(config_cmd.app, name="config")
 from .commands.run import run as _run_cmd
```

Ruff (0.14.3)

Redefinition of unused `_run_cmd` from line 10

Remove definition: `_run_cmd`

(F811)